### ISSUE TITLE
undefined reference to `_snd_module_control_empty'

### ISSUE THREAD

**Issue by madebr** [alsa-project/alsa-lib#168]
When linking `aserver` of libalsa 1.2.5.1,
the following linking error appears:
```
/usr/bin/ld: ../src/.libs/libasound.a(control_symbols.o):(.data.rel+0x8): undefined reference to `_snd_module_control_empty'
collect2: error: ld returned 1 exit status
```

libalsa is built as a static library.

Full log:
[libalsa.log](https://github.com/alsa-project/alsa-lib/files/6951192/libalsa.log)
(this is using the conan build recipe at https://github.com/conan-io/conan-center-index/tree/master/recipes/libalsa/all)

Thanks

**Comment by perexg** [comment]
Already fixed in 81e7923fbfad45b2f353a4d6e3053af51f5f7d0b .

**Comment by madebr** [comment]
Thanks for the answer!
